File: bbclient/src/com/bbclient/IsEmptyRunnable.java
package com.bbclient;

import com.hazelcast.map.IMap;

class IsEmptyRunnable extends IMapMethodRunnable {
    IsEmptyRunnable(IMap<Integer, String> map) {
        super(map,"isEmpty", false);
    }

    void invokeMethod() { // ignore newValue
        var isEmpty = map.isEmpty();
        assert(!isEmpty);
    }
}

File: bbclient/src/com/bbclient/IsEmptyRunnable.kt
package com.bbclient

import com.hazelcast.map.IMap

internal class IsEmptyRunnable(map: IMap<Int?, String?>) : IMapMethodRunnable(map, "isEmpty", false) {
    override fun invokeMethod() { // ignore newValue
        val isEmpty = map.isEmpty
        assert(!isEmpty)
    }
}
